rust/arrow/src/datatypes.rs

```diff
@@ -1594,7 +1620,7 @@ impl Field {
 
 impl fmt::Display for Field {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "{}: {:?}", self.name, self.data_type)
+        write!(f, "{:?}", self)
```

Comment:
This defaults us to `Display` which we could `#[derive(Display)]` instead. I think we could remove this function in that case.

Reply:
`#[derive(Display)]` does not work, are suggesting https://crates.io/crates/derive_more ?
Anyway, I tend to leave this change to a following PR.

Reply:
Ah, yes, I was thinking of `#[derive(Debug)]`